ecl/eclagent/eclagent.cpp

```diff
@@ -1980,6 +1980,14 @@ void EclAgent::doProcess()
         const cost_type diskAccessCost = aggregateDiskAccessCost(w, nullptr);
         if (diskAccessCost)
             w->setStatistic(queryStatisticsComponentType(), queryStatisticsComponentName(), SSTglobal, "", StCostFileAccess, NULL, diskAccessCost, 1, 0, StatsMergeReplace);
+        stat_type totalSizeSpill = 0, peakSizeSpill = 0;
```

Comment:
>This means the global values will not be updated until the workunit completes. It is likely to be useful if this was updated while it runs

Made same point in previous comment here : https://github.com/hpcc-systems/HPCC-Platform/pull/16845#discussion_r1088167357

But thought it could be tackled as a separate/next JIRA/PR.

Reply:
I'll work on this in a separate Jira.